Source/DafnyLanguageServer/CounterExampleGeneration/DafnyModel.cs

```diff
@@ -54,27 +54,27 @@ public class DafnyModel {
         _ => 0
       };
       fSetSelect = ModelFuncWrapper.MergeFunctions(this, new List<string> { "MapType0Select", "MapType1Select" }, 2);
-      fSeqLength = new ModelFuncWrapper(this, "Seq#Length", 1, tyArgMultiplier);
-      fSeqBuild = new ModelFuncWrapper(this, "Seq#Build", 2, tyArgMultiplier);
-      fSeqAppend = new ModelFuncWrapper(this, "Seq#Append", 2, tyArgMultiplier);
-      fSeqDrop = new ModelFuncWrapper(this, "Seq#Drop", 2, tyArgMultiplier);
-      fSeqTake = new ModelFuncWrapper(this, "Seq#Take", 2, tyArgMultiplier);
-      fSeqIndex = new ModelFuncWrapper(this, "Seq#Index", 2, tyArgMultiplier);
-      fSeqUpdate = new ModelFuncWrapper(this, "Seq#Update", 3, tyArgMultiplier);
+      fSeqLength = new ModelFuncWrapper(this, "Seq#Length", 1, 0);
```

Comment:
What is the tyArgMultipler?

Reply:
It specifies how many arguments exist in the SMT-Lib function definitions for each type argument. At the moment, it's either 0 or 1, depending on the value of `TypeEncoding`. This seemed easier than having conditionals everywhere.